### DISCUSSION TITLE
Fuji solar inverter 1P

### DISCUSSION THREAD

**Discussion by krmsi** [davidrapan/ha-solarman#155] [Q&A] [answered]
Good day,

Does anybody knows what inverter template you are using for Fuji solar inverters? I can see some information's using deye_hybrid but some of them are still missing?

Thank you in advance.

**Comment by davidrapan** [comment]
Hi @krmsi, I'm not really able to help you much with this but if you maybe could list all the entities which does work with mentioned profile and also provide some screenshots from integration and also the solarman app you are using (to easily identify correct values)? So I can create separate profile for your device w/o entities which does not work and we can start from there... 😉

**Comment by dcigir** [comment, marked as answer]
I'm using deye_string.yaml

**Reply by davidrapan** [reply]
That means that auto profile selection should also work?

**Reply by dcigir** [reply]
Yes
